README.md

```diff
@@ -149,6 +149,12 @@ tenant:
   # https://grafana.com/docs/mimir/latest/configure/about-tenant-ids/
   # env: CT_TENANT_PREFIX
   prefix: foobar-
+
+  # If true will use the tenant ID of the inbound request as the prefix of the new tenant id.
```

Comment:
I think we need a bit more detailed description, maybe with a example etc. I didn't get it at first :) Maybe clarify that it takes X-Org-Id header from incoming request, concatenates etc

Reply:
I've added an example inline. Do you think that's sufficient? If not I can add a more detailed example further down in the README.